src/components/button/Button.ts

```diff
@@ -0,0 +1,37 @@
+import Block from '../../core/block/Block';
+import Loader from '../loader/Loader';
+
+export default class Button extends Block {
+  render() {
+    let cl;
+    if (this.props.className) {
+      cl = this.props.className;
+    } else {
+      cl = '';
+    }
```

Comment:
Такую запись можно сократить
```suggestion
    const className = this.props.className ?? '';
```
либо
```suggestion
    const {className = ''} = this.props;
```
и лучше давать более понятные названия переменным